firebase-project/src/components/navbar.vue

```diff
@@ -38,7 +39,19 @@ export default {
       return router.currentRoute.value.path === "/login";
     });
 
-    return { isLoginPage };
+    const isPostRoute = computed(() => {
```

Comment:
this is totally wrong method to implement it.
You should check for the user data or token @mukeshsin